src/d3/d3Renderer.js

```diff
@@ -502,9 +569,17 @@ var d3Renderer = function (arg) {
     }
```

Comment:
should we return m_this here as well?

Reply:
This is called by `renderAnimationFrame`, so any return value will be ignored.